src/shared/components/community/community-link.tsx

```diff
@@ -21,20 +21,19 @@ export class CommunityLink extends Component<CommunityLinkProps, any> {
 
   render() {
     const community = this.props.community;
-    let name_: string, title: string, link: string;
+    let title: string, link: string;
     const local = community.local === null ? true : community.local;
+    const domain = hostname(community.actor_id);
     if (local) {
-      name_ = community.name;
       title = community.title;
       link = `/c/${community.name}`;
     } else {
-      const domain = hostname(community.actor_id);
-      name_ = `${community.name}@${domain}`;
+      const name_ = `${community.name}@${domain}`;
       title = `${community.title}@${domain}`;
       link = !this.props.realLink ? `/c/${name_}` : community.actor_id;
     }
 
-    const apubName = `!${name_}`;
+    const apubName = `!${community.name}@${domain}`;
```

Comment:
This *should* be okay, because it should only use the apubName on community pages, hopefully not in most community links.